resources/disksetup.rb

```diff
@@ -0,0 +1,34 @@
+# Resource properties
+property :disks, Array, required: true
+property :stripe_width, Integer
+property :tmp_disk_file, String, default: '/tmp/disks.txt'
```

Comment:
I'd suggest `Tempfile.new(['disks', 'txt'])`

Reply:
I don't want to randomize the tmp filename here. There is no gain to do it.

Reply:
If there more than one resource how do you handle that?